lazer/sdk/rust/client/Cargo.toml

```diff
@@ -6,7 +6,7 @@ description = "A Rust client for Pyth Lazer"
 license = "Apache-2.0"
 
 [dependencies]
-pyth-lazer-protocol = "0.5.0"
+pyth-lazer-protocol = { path = "../protocol", version = "0.5.0" }
```

Comment:
doesn't this break the publishing to cargo?

Reply:
No, cargo is smart enough to remove paths on publish.